### PR TITLE
Added default value for asserts validation

### LINKED ISSUE #1234: Validation does not work
Validation is not working for Default groups.

after change this line: https://github.com/sonata-project/SonataUserBundle/blob/b3b54effcd225194b48ff752e90e4c4567b31f64/src/Admin/Model/UserAdmin.php#L46

default Assert back to work.

change: 

```php
$options['validation_groups'] = (!$this->getSubject() || null === $this->getSubject()->getId()) ? ['Default', 'Registration'] : ['Default', 'Profile'];
```

### PR DESCRIPTION
**Pull request by eerison**
## Default validation group to UserModel

I am targeting this branch Because the last version of SonatauserBundle is not working for the Default group.

Fix #1234.

## Changelog

```markdown
### Fixed
- Fixed Asserts validation for Default group.
```

### CODE REVIEW

**Comment by eerison** [comment]
could you add label

#hacktoberfest-accepted

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### Fixed
```
Comment (core23) [inline CHANGELOG.md:5]: Please revert this. The file is auto-generated

**Review by core23 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (eerison) [reply CHANGELOG.md:5]: ohhh sorry I didn`t now this

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Do you use an old branch ? Tests are failing.

**Comment by eerison** [comment]
@VincentLanglet maybe yes, But I updated now from 4.x and it`s broken 2 tests yet.

**Comment by eerison** [comment]
well no make sense, I created another branch from 4.x and now it`s failing more tests 😕

**Comment by VincentLanglet** [comment]
@eerison This is the reason: https://github.com/sonata-project/SonataUserBundle/pull/1250

**Comment by eerison** [comment]
@VincentLanglet cool, it works thank you :)

**Comment by VincentLanglet** [comment]
Please review @core23

**Review by core23 — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks @eerison

**Comment by eerison** [comment]
@VincentLanglet anyide when will be the next release? sonata project are the some roadmap or something like that?

**Comment by VincentLanglet** [comment]
@eerison I'm on it, https://github.com/sonata-project/SonataUserBundle/pull/1264

**Pull request merged**